packages/@react-spectrum/illustratedmessage/docs/IllustratedMessage.mdx

```diff
@@ -0,0 +1,97 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/illustratedmessage';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+import {Content} from '@react-spectrum/view';
+import {Heading} from '@react-spectrum/typography';
+import {IllustratedMessage} from '@react-spectrum/illustratedmessage';
+```
+
+# IllustratedMessage
+
+<p>{docs.exports.IllustratedMessage.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['IllustratedMessage']} />
+
+## Example
+
+```tsx example
+<IllustratedMessage>
+  <svg width="150" height="103" viewBox="0 0 150 103" aria-hidden="true" role="presentation">
+    <path d="M133.7,8.5h-118c-1.9,0-3.5,1.6-3.5,3.5v27c0,0.8,0.7,1.5,1.5,1.5s1.5-0.7,1.5-1.5V23.5h119V92c0,0.3-0.2,0.5-0.5,0.5h-118c-0.3,0-0.5-0.2-0.5-0.5V69c0-0.8-0.7-1.5-1.5-1.5s-1.5,0.7-1.5,1.5v23c0,1.9,1.6,3.5,3.5,3.5h118c1.9,0,3.5-1.6,3.5-3.5V12C137.2,10.1,135.6,8.5,133.7,8.5z M15.2,21.5V12c0-0.3,0.2-0.5,0.5-0.5h118c0.3,0,0.5,0.2,0.5,0.5v9.5H15.2z M32.6,16.5c0,0.6-0.4,1-1,1h-10c-0.6,0-1-0.4-1-1s0.4-1,1-1h10C32.2,15.5,32.6,15.9,32.6,16.5z M13.6,56.1l-8.6,8.5C4.8,65,4.4,65.1,4,65.1c-0.4,0-0.8-0.1-1.1-0.4c-0.6-0.6-0.6-1.5,0-2.1l8.6-8.5l-8.6-8.5c-0.6-0.6-0.6-1.5,0-2.1c0.6-0.6,1.5-0.6,2.1,0l8.6,8.5l8.6-8.5c0.6-0.6,1.5-0.6,2.1,0c0.6,0.6,0.6,1.5,0,2.1L15.8,54l8.6,8.5c0.6,0.6,0.6,1.5,0,2.1c-0.3,0.3-0.7,0.4-1.1,0.4c-0.4,0-0.8-0.1-1.1-0.4L13.6,56.1z" />
+  </svg>
+  <Heading>No Results</Heading>
+  <Content>Try another search</Content>
+</IllustratedMessage>
+```
+
+## Content
+
+Each of these sections can be populated by providing the following components to your IllustratedMessage as children:
+`svg`, `Heading` (title), and `Content` (body).
```

Comment:
Maybe include a sentence/diagram before this stating something to the effect of "A IllustratedMessage consists of three areas: a illustration, a title, and  a body." The current sentence feels a bit off since there were no section mentioned prior.